repo/packages/H/hivemq/0/resource.json

```diff
@@ -0,0 +1,60 @@
+{
+    "assets": {
+        "uris": {
+            "jre-tar-gz": "https://downloads.mesosphere.com/java/server-jre-8u162-linux-x64.tar.gz", 
+            "libmesos-bundle-tar-gz": "https://downloads.mesosphere.com/libmesos-bundle/libmesos-bundle-1.11.0.tar.gz", 
+            "bootstrap-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.55.4/bootstrap.zip", 
+            "executor-zip": "https://downloads.mesosphere.com/dcos-commons/artifacts/0.55.4/executor.zip", 
```

Comment:
hi @sbaier1 can you remove the executor, these days the mesos default executor gets used. This currently makes one of the universe tests fail.

cc @ryadav88

Reply:
tested it locally and updated the PR accordingly